e2e/paths/uploadDocuments_test.js

```diff
@@ -29,3 +29,15 @@ Scenario('Uploading all files in the c110a application', (I, uploadDocumentsPage
   I.seeDocument('Social work assessment', 'mockFile.txt', 'Attached');
   I.seeDocument('Care plan', 'mockFile.txt', 'Attached');
 });
+
+Scenario('As a local authority I have the ability to upload a document after submission of a case', (I, uploadDocumentsPage, caseViewPage) => {
+  uploadDocumentsPage.selectSocialWorkChronologyToFollow(config.testFile);
+  I.continueAndSubmit();
+  caseViewPage.goToNewActions(config.applicationActions.submitCase);
+  I.click('Submit');
+  I.waitForElement('.tabs');
+  caseViewPage.goToNewActions(config.applicationActions.uploadDocuments);
+  uploadDocumentsPage.selectSocialWorkChronologyToFollow(config.testFile);
```

Comment:
It would be good to upload other document after submission and assert that that additional document exists in evidence tab.